internal/awsservice/cloudwatchlogs.go

```diff
@@ -6,16 +6,14 @@ package awsservice
 import (
 	"context"
 	"errors"
+	"github.com/qri-io/jsonschema"
```

Comment:
Why? It's in the go mod

Reply:
[Goimport](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) for formatting and grouping the similar dependencies together. VSCodes has `go-group-imports` that [groups by default](https://marketplace.visualstudio.com/items?itemName=aleksandra.go-group-imports) with 
```
Standard library imports,
Third party imports,
Own (belonging to the organization) imports.
```

Reply:
Is this part of our linter?

Reply:
The linter doesn't run because the Makefile is invalid. I fixed that and there are lots of linter errors. I'm not fixing those in this PR because it's already a month old. I moved the import manually.